### ISSUE TITLE
Build and packaging for archlinux / manjaro

### ISSUE THREAD

**Issue by genofire** [llewelld/harbour-contrac#74]
Create a PKGBUILD and upload it into https://aur.archlinux.org

so it become useable on [pinephone](https://www.pine64.org/pinephone/) for [manjaro](https://wiki.pine64.org/index.php?title=PinePhone_Software_Releases#Manjaro_ARM) and [archlinux](https://github.com/dreemurrs-embedded/Pine64-Arch) and other linux-phone (without sailfish)

**Comment by bionade24** [comment]
Isn't there any official package registry for Manjaro ARM? Additionally, the problem is that on Arch or Manjaro libsilica is not available. So maybe check if this program was already ported to Ubuntu touch first, but otherwise you would have to use diffrent Qt coponents on some parts.

**Comment by bionade24** [comment]
The problem is that this thing relies on libsilica. Is libsilica available on Manjaro ARM? Probably this thing has to be ported and some Qt components have to be replaced. Maybe there already is a port for Ubuntu touch?

**Comment by llewelld** [comment]
Yeah, I really like the idea of this being usable more widely, but it's a good point about the lack of Silica.

It probably wouldn't be too hard to create a very simple pure-Qt UI that could then be embellished for different distros. If I get the chance, or if someone else is up for it, then I agree it would be worthwhile.

**Comment by bionade24** [comment]
> Yeah, I really like the idea of this being usable more widely, but it's a good point about the lack of Silica.
> 
> It probably wouldn't be too hard to create a very simple pure-Qt UI that could then be embellished for different distros. If I get the chance, or if someone else is up for it, then I agree it would be worthwhile.

I PMed @genofire on a diffrent platform and offered to meet at rc3.world (I pretty sure @genofire participates, too) so I could explain the basics if wanted. Maybe this persons wants to pick this up.

**Comment by llewelld** [comment]
Thanks for taking the initiative @bionade24, that's really great. And If @genofire is interested then that would also be superb. Any way I can support I'd be happy to.

**Comment by llewelld** [comment]
Blocked by #80.

**Comment by bionade24** [comment]
Maybe we could copy the approach of: https://github.com/rinigus/pure-maps?
